.travis.yml

```diff
@@ -14,3 +16,4 @@ script:
   - make verify
   - make smb-windows  
   - make container
+  - make integration-test
```

Comment:
I think you could add `sudo` here, refer to:
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/.travis.yml#L12